lib/sentry/config.ex

```diff
@@ -388,6 +388,14 @@ defmodule Sentry.Config do
       The number of lines of source code
       before and after the line that caused the exception to report.
       """
+    ],
+    otp_apps: [
```

Comment:
I think we can find a better name for this, because `otp_apps` doesn't really tell you what the OTP apps you're providing are for, no?

Maybe `in_app_otp_apps`?